src/skin/qml/qmllibraryproxy.cpp

```diff
@@ -0,0 +1,25 @@
+#include "skin/qml/qmllibraryproxy.h"
+
+#include <QAbstractItemModel>
+
+#include "library/library.h"
+#include "skin/qml/qmllibrarytracklistmodel.h"
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+QmlLibraryProxy::QmlLibraryProxy(Library* pLibrary, QObject* parent)
+        : QObject(parent),
+          m_pLibrary(pLibrary),
+          m_pModel(
```

Comment:
Why not pass `this` as `parent` and use `parented_ptr` for the member as suggested? Object hierarchies are what Qt is supposed to manage by parenting.

Reply:
I thought this wouldn't work due to the Q_PROPERTY, but it does :tada: eff29a67989b95e45b0c5de5faaa8fb1dfe4a6f3